src/mesh/mesh.cpp

```diff
@@ -29,6 +29,9 @@
 #include "../athena.hpp"
 #include "../athena_arrays.hpp"
 #include "../bvals/bvals.hpp"
+#include "../bvals/sixray/bvals_sixray.hpp" //SixRayBoundaryVariable
```

Comment:
I would prefer to delete the trailing comments after Athena++ header includes

Reply:
deleted